projects/authService/intro.md

```diff
@@ -1,12 +1,18 @@
 ---
-title: HPCC Auth Service
-shortDescription: At its core, Auth-service relies on JWT tokens for robust access control, employing digital signatures and key pair verification to ensure that only authorized users can enter protected areas. Beyond token management, Auth-service offers an intuitive user interface for seamless user administration, enabling organizations to easily manage user accounts and permissions.
+title: HPCC Systems® Auth Service
+shortDescription: Auth-service is a robust access control system, ensuring secure access to protected areas through digital signatures and key pair verification. It simplifies user administration with an intuitive interface, allowing organizations to effortlessly manage user accounts and permissions, enhancing overall security and access management. 
 gitHubRepo: https://github.com/hpcc-systems/Auth-Service
 imageName: auth-service
 ---
 
-In the realm of digital security and user authentication, Auth-service shines as a trusted solution, utilized by instances of Tombolo and REAL-BI, and tested with HPCC clusters for user authentication and authorization.
+In the realm of digital security and user authentication, Auth-service shines as a trusted solution, utilized by instances of Tombolo and Real-BI, and  tested with HPCC clusters for user authentication and authorization.
```

Comment:
In another file you use "Real BI" rather than "Real-BI".  Which is it?  Can we go through the entire site and make sure its name is consistent?